### DISCUSSION TITLE
Set SplashScreen animation color

### DISCUSSION THREAD

**Discussion by minghwee** [timbru31/cordova-plugin-lottie-splashscreen#463] [General]
Hi

Can I check if we can specify the Lottie animation color when it is in dark mode? (I would like the splash screen when it is in dark mode to look that same as light mode)
i.e. I would like the splash screen In the dark mode and light mode to look the same.

thank you

Best Regards

**Comment by timbru31** [comment]
You can set the background color differently for light and dark mode. If you want a different animation color of the animation itself, I'd recommend making two Lottie animations and then use them via `LottieAnimationLocationDark` and `LottieAnimationLocationLight`.

If you want it to be the same, just use `LottieAnimationLocation`.
Does that answer your question?

**Reply by minghwee** [reply]
Hi @timbru31 , thank you for the reply. But for certain android like Xiaomi https://www.xiaomist.com/2020/04/global-dark-mode-this-is-how-miui-12.html, the OS seems to have a forced dark mode which inverts all the colors, including the Splashscreen, and I had tried what u had shared earlier and it doesn't seems to work as expected. ( I'm seeing a dark Lottie splash screen instead of a light splash screen).

**Reply by timbru31** [reply]
Thanks for the follow up details. I need to look into the MUI specifics, as it seems Xiaomi does the inverting themselves. :/

**Reply by minghwee** [reply]
@timbru31 thanks for the response!
